ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_dev/InterfaceIT.scala

```diff
@@ -0,0 +1,104 @@
+package com.daml.ledger.api.testtool.suites.v1_dev
+
+import com.daml.error.definitions.LedgerApiErrors
+import com.daml.ledger.api.testtool.infrastructure.Allocation.{
+  Participant,
+  Participants,
+  SingleParty,
+  allocate,
+}
+import com.daml.ledger.api.testtool.infrastructure.Assertions._
+import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
+import com.daml.ledger.api.testtool.infrastructure.TransactionHelpers._
+import com.daml.ledger.api.v1.value.Identifier
+import com.daml.ledger.client.binding.Primitive
+import com.daml.ledger.test.semantic.Interface._
+import scalaz.Tag
+
+class InterfaceIT extends LedgerTestSuite {
```

Comment:
@sofiafaro-da Do you have idea what we should test here ?

Reply:
The current implementation does not offer type checking feature. 
It is a separate bullet #13653